services/twitter.py

```diff
@@ -21,13 +21,20 @@ def plugin(srv, item):
     )
 
     text = item.message[0:138]
+    try:
+        text = text.decode('utf-8')
+    except Exception,  err:
+        srv.logging.debug("Cannot convert to unicode: %s" % (str(err)))
```

Comment:
are you sure this message is correct? that you are converting TO unicode?

Reply:
I think i understand why you questioning that. I meant converting to unicode string from string which contains non-ascii, it would be 'changing' meaning rather
As i remember 'text' had issues with non-ascii chars as a string.